### ISSUE TITLE
peer2profit & packetStream issue

### ISSUE THREAD

**Issue by crishin** [chashtag/PiCash#18]
Hi,

peer2profit is no longer available as docker image (confirmed by Peer2Profit Support Chat in Telegram).

PacketStream has the client not updated and need to run as docker container independently for works fine.

**Comment by jaypathak9460** [comment]
I am trying to go to peer2Profit and it says no website found 404 error I have done this before without getting an error but when I am setting it up now it shows me an error like No such container pi cash after waiting 30 sec for registering the for earn app

**Comment by crishin** [comment]
Yes because now peer2profit is only available on Telegram Bot. In the previous months when you tried to access in p2p web page, a disclaimer figured out to inform about this... about the client, only android, Linux and macos is now supported... regarding Linux, not for rpi arm64

> I am trying to go to peer2Profit and it says no website found 404 error I have done this before without getting an error but when I am setting it up now it shows me an error like No such container pi cash after waiting 30 sec for registering the for earn app
> 
>

**Comment by isotisot** [comment]
Peer2Profit is a scam, needs to be removed from this project.  Don't trust this project until it's removed.
